packages/core/src/services/datasetRows/createRowsFromUploadedDataset.ts

```diff
@@ -129,13 +133,19 @@ export async function createRowsFromUploadedDataset(
   })) {
     if (batch === null) {
       onFinished?.()
-      const deleteResult = await disk.delete(fileKey)
 
-      if (deleteResult.error) {
-        throw new Error(
-          `Error deleting file: ${deleteResult.error.message} datasetId: ${datasetId} workspaceId: ${workspaceId}`,
-        )
-      }
+      if (!deleteFile) break
+
+      // PARANOID MODE: I don't want to enable file deletion until we have migrated
+      // all datasets V1
+
+      /* const deleteResult = await disk.delete(fileKey) */
+      /**/
+      /* if (deleteResult.error) { */
+      /*   throw new Error( */
+      /*     `Error deleting file: ${deleteResult.error.message} datasetId: ${datasetId} workspaceId: ${workspaceId}`, */
+      /*   ) */
+      /* } */
```

Comment:
This service is used in new datasets V2 settings to generate rows in `dataset_rows` table out of the uploaded CSV by the user. 

I'm reusing all that logic for the migration from dataset V1 to V2. Because V1 datasets have stored in Disk/S3 the file and I pass that file to this service and everything is created in the new way as it should be. 

The only thing I don't want is to delete the file after the new data is created. Because of that I commented this code until we're fully migrated.